shortcircuit-api/src/main/java/com/powsybl/shortcircuit/SimpleShortCircuitBusResults.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.shortcircuit;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class SimpleShortCircuitBusResults extends AbstractShortCircuitBusResults {
```

Comment:
getInitialVoltageMagnitude() could be also implemented here and retrieve from voltage andvoltage drop, no?

Reply:
Actually it is in the results, I just forgot to put it.